02-linked-lists/02-linked-lists-implementation.md

```diff
@@ -741,7 +750,8 @@ In the nominal case, where the list already has some number of nodes, consider u
 
 
 Still feeling stuck? Check this video walkthrough of the solution.
-<!-- TODO: Add video walkthrough to explanation -->
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=60f22d0f-5fc5-4054-9cc0-aef6014384ac&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="360" width="640" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
```

Comment:
Just a note that you are cut out of the frame a bit here, but otherwise this is a really good explanation!